man/make.conf.5

```diff
@@ -405,6 +405,12 @@ exist). Also see the related \fIunmerge\-backup\fR feature.
 Use locks to ensure that unsandboxed ebuild phases never execute
 concurrently. Also see \fIparallel\-install\fR.
 .TP
+.B ebuildshell
```

Comment:
I think it makes more sense to make this an ebuild(1) commandline option rather than a FEATURES setting, since FEATURES settings tend to be used much differently.  An environment variable can be used to signal to various code that ebuildshell is active.

Reply:
Not sure if I fully understand this: Do you suggest something like `emerge --ebuildshell <atom>` instead of `FEATURES=ebuildshell emerge <atom>` for front end usage?
(I won't care, but I want to be able to get the ebuildshell prompt for each phase with the `emerge` command.)